components/modal/modal.vue

```diff
@@ -288,6 +302,32 @@ export default {
   },
 
   computed: {
+    modalListeners () {
+      return {
+        click: event => {
+          (event.target === event.currentTarget) && this.close();
+          this.$emit('click', event);
+        },
+
+        keydown: event => {
+          switch (event.code) {
+            case EVENT_KEYNAMES.esc:
+            case EVENT_KEYNAMES.escape:
```

Comment:
Would it ever be "esc" rather than "escape" when using code? https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values

Reply:
Looks like very old versions of IE used "esc". Guess it doesn't hurt to keep it.

Reply:
Yeah, I've used the same mapping [Vue is using](https://github.com/vuejs/vue/blob/d6bdff890322bc87792094a1690bcd16373cf82d/src/compiler/codegen/events.ts#L21-L36) for the alias of these events and some of them are for IE11 support.